src/plugins/intel_gpu/src/graph/convolution.cpp

```diff
@@ -382,21 +365,12 @@ convolution_inst::typed_primitive_inst(network& network, convolution_node const&
                               "expected output size",
                               1,
                               "Only one-dimensional batch size are supported");
-        CLDNN_ERROR_LESS_THAN(node.id(),
+        CLDNN_ERROR_NOT_EQUAL(node.id(),
```

Comment:
Are you sure that this is correct macro for all group cases (when group is equal to 1 and when group isn't equal to 1)? @sshlyapn what do you think?

Reply:
I think yes. At least all tests pass with such check

Reply:
As for me the condition looks correct